server/mvc/controllers/ProfileController.js

```diff
@@ -9,7 +9,7 @@ exports.info = function(req, res){
   user
     .findByUserId(req.user.id)
     .then(function(){
-      if (!user.exists()) {
+      if (!user.existsInDatabase()) {
         throw new Error('User does not exit');
```

Comment:
Well. It's temporary code and will be removed eventually. I'm just gonna leave it...